### ISSUE TITLE
MAC address requires colons

### ISSUE THREAD

**Issue by mr-brunes** [basildane/WakeOnLAN#145]
I struggled with trying to add a new host and the OK button stubbornly remained greyed out, until I found that it was requiring me to have colons between the MAC address octets.
Can this strict requirement be removed as it'd make copy and pasting in addresses easier? For example Windows uses dashes as separators and these don't work either.

**Comment by basildane** [comment]
Hi, I'm looking at that function, it does allow colons, dashes, or MAC addresses with just a simple 12 character string.  I tested it with all three options.  Perhaps there was another field you did not have filled in that was keeping the OK button greyed out?  Maybe FQDN or IP address?

**Comment by mr-brunes** [comment]
![image](https://user-images.githubusercontent.com/1270750/56360494-ba6e4900-61dc-11e9-8cf8-8f30e0b25d5d.png)
Still greyed out for me with dashes.

**Comment by basildane** [comment]
and what happens when you move the cursor to the next field, causing it to validate the mac address?

**Comment by mr-brunes** [comment]
Aha just found that if I switch to another tab and move back the OK button is available.

**Comment by basildane** [comment]
You don't have to switch tabs, just move to another field.

**Comment by mr-brunes** [comment]
Ok that works - odd how I never seemed to trigger that before!

**Comment by basildane** [comment]
Yes, I don't like how that validate function works either.  I'll try and clean that up when I get time.

**Comment by mr-brunes** [comment]
It's weird that I've not seen that type of entry validation before. Athough in hindsight it makes sense on paper, it doesn't seem to when you use it, IYSWIM. ;-)
Thanks for looking anyway.
